src/01-functions.js

```diff
@@ -93,7 +108,11 @@ function average() {}
  *  averageThree(10, 11, 19);
  *  //> 10 // 10 + 11 + 19 = 30; 30 / 3 = 10;
  */
-function averageThree() {}
+function averageThree(a, b, c) {
+  let result = (a + b + c) / 3
+  return result;
+
```

Comment:
Let's get in the habit of removing empty whitespace lines if they are not needed or are breaking the logic in a particular spot that is good to break the logic.